.github/workflows/dev_deploy.yml

```diff
@@ -5,10 +5,55 @@ on:
     branches: [ 'dev' ]
 
 jobs:
-  deploy:
+  push_to_registry:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: all
+
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: |
+          slatedocs/slate
+        tags: |
+          type=ref,event=branch
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_ACCESS_KEY }}
+
+    - name: Push to Docker Hub
+      uses: docker/build-push-action@v2
+      with:
+        builder: ${{ steps.buildx.outputs.name }}
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64,linux/arm64,linux/ppc64le
+        push: true
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
```

Comment:
why are we pushing this to docker hub? who's dockerhub account is this? If we need to store a Docker image, putting it on AWS ECR may be simpler in the long run as we might be removing stuff from Ada's Dockerhub account

(though ECR may make running this locally harder than hosting a public docker image, though AWS does have public ECR repos now)

Reply:
we're not pushing to docker hub
this is from the upstream repo and would only run on a branch called `dev`